### ISSUE TITLE
Implement trained resnet model

### ISSUE THREAD

**Issue by deepiksdev** [deepiksdev/torch#12]
See https://github.com/facebook/fb.resnet.torch/tree/master/pretrained

**Comment by Lolitawxc** [comment]
@deepiksdev  I installed a working Resnet 200 model at ami-3f43d228
To classify an image, just do:
```
cd fb.resnet.torch/pretrained
th classify.lua resnet-200.t7 my_photo.jpg 
```

**Comment by deepiksdev** [comment]
@Lolitawxc 

> I installed a working Resnet 200 model at ami-3f43d228

Are the results OK? Please show some examples.

Additionally, if the results are OK, please note that we want to use a single server for all issues in this repository, and we want to be able to have a http server (see #3) . Please organize and publish your code accordingly to this repo.

**Comment by Lolitawxc** [comment]
@deepiksdev 
> Are the results OK? Please show some examples.

Not really. I mean it recognizes well the objects but for almost exact same images it does
not tag the same (even for high accuracy classification) which is a big problem for us.
With the images of move publishing, the first 2 images are well labelled, but not the other 3.

### Move publishing:

![_ak13916](https://cloud.githubusercontent.com/assets/12127944/17323760/9887cb12-58a3-11e6-9edd-4f2a24344725.jpg)
```
0.99983048439026	disk brake, disc brake	
0.00013200745161157	car wheel	
1.0521655894991e-05	chain	
5.1939841796411e-06	cannon	
3.8650750866509e-06	mountain bike, all-terrain bike, off-roader	
```

![_ak13919](https://cloud.githubusercontent.com/assets/12127944/17323777/a4114a08-58a3-11e6-88f8-7309c66a8baa.jpg)
```
Classes for	_AK13919.jpg	
0.99994647502899	disk brake, disc brake	
3.8988764572423e-05	car wheel	
7.4000686254294e-06	chain	
2.5093327167269e-06	mountain bike, all-terrain bike, off-roader	
1.1310588661217e-06	moped	
```

![_ak13985](https://cloud.githubusercontent.com/assets/12127944/17323784/ad37a21c-58a3-11e6-9c66-c2f0911fe6bb.jpg)
```
Classes for	_AK13985.jpg	
0.19119782745838	crash helmet	
0.1276320964098	moped	
0.097405396401882	military uniform	
0.065067730844021	cannon	
0.047406315803528	mountain bike, all-terrain bike, off-roader	
```

![_ak13986](https://cloud.githubusercontent.com/assets/12127944/17323806/ca7f3682-58a3-11e6-8d25-192469718e22.jpg)
```
Classes for	_AK13986.jpg	
0.67302292585373	bicycle-built-for-two, tandem bicycle, tandem	
0.05692595988512	mountain bike, all-terrain bike, off-roader	
0.033735010772943	castle	
0.033064093440771	unicycle, monocycle	
0.02350570820272	parachute, chute	
```

![_ak13991](https://cloud.githubusercontent.com/assets/12127944/17323809/d05e9926-58a3-11e6-874a-afc2a305f66c.jpg)
```
Classes for	_AK13991.jpg	
0.40418618917465	moped	
0.16411550343037	bicycle-built-for-two, tandem bicycle, tandem	
0.046110033988953	motor scooter, scooter	
0.034014582633972	parachute, chute	
0.031058494001627	mountain bike, all-terrain bike, off-roader	
```

**Comment by deepiksdev** [comment]
@Lolitawxc 

>  I mean it recognizes well the objects but for almost exact same images it does
not tag the same (even for high accuracy classification) which is a big problem for us.
With the images of move publishing, the first 2 images are well labelled, but not the other 3.

Interesting comment. Can you test with our existing tagging service using Microsoft and Google?

**Comment by Lolitawxc** [comment]
@deepiksdev 
When I give the bot my image I get 'There seems to be a problem...'

**Comment by deepiksdev** [comment]
@Lolitawxc 

>> Interesting comment. Can you test with our existing tagging service using Microsoft and Google?

> When I give the bot my image I get 'There seems to be a problem...'

You don't need to use the bot for that.

Additionally, can you test with the Inception V3 microservice we had implemented?

**Comment by Lolitawxc** [comment]
@deepiksdev 
> Additionally, can you test with the Inception V3 microservice we had implemented?

Already tried yesterday and it does not work.
Because we always take the same image (that was given in the package).
The microservice does not take argument.
I have to remake a package to take arguments but since I don't have the initial package I used I fear it will make too much time to redo the package for what it's worth.

**Comment by deepiksdev** [comment]
@Lolitawxc 

> since I don't have the initial package I used

Isn't it possible to download it from lambda? Isn't it available in the repository?

**Comment by Lolitawxc** [comment]
@deepiksdev 

Isn't it possible to download it from lambda? Isn't it available in the repository?

I just find a way to do it. I will upload the results of v3 later in the day.

**Comment by Lolitawxc** [comment]
@deepiksdev 
> can you test with the Inception V3 microservice we had implemented?

I think inception has the same problem.

For this duplicated photo,

![_ak13916](https://cloud.githubusercontent.com/assets/12127944/17396167/631bca88-5a32-11e6-920b-e588460d90f0.jpg)

I got:

### Img1

```
"disk brake, disc brake (score = 0.93541)motor scooter, scooter (score = 0.00307)car wheel (score = 0.00162)moped (score = 0.00112)chain (score = 0.00056)"
```

### Img2

```
"disk brake, disc brake (score = 0.94041)motor scooter, scooter (score = 0.00604)moped (score = 0.00337)car wheel (score = 0.00273)chain (score = 0.00076)"
```

For this duplicated photo,

![_ak13985](https://cloud.githubusercontent.com/assets/12127944/17396406/9171786e-5a33-11e6-8d7a-c2fb722726f2.jpg)

### Img1

```
"castle (score = 0.15686)palace (score = 0.10575)monastery (score = 0.09062)triumphal arch (score = 0.05245)lakeside, lakeshore (score = 0.03862)"
```

### Img 2

```
"monastery (score = 0.20079)alp (score = 0.10370)castle (score = 0.09811)stupa, tope (score = 0.03701)triumphal arch (score = 0.03693)"
```

### Img 3

```
"bicycle-built-for-two, tandem bicycle, tandem (score = 0.21061)mountain bike, all-terrain bike, off-roader (score = 0.11594)lakeside, lakeshore (score = 0.04946)castle (score = 0.04353)monastery (score = 0.04145)"
```

**Comment by deepiksdev** [comment]
OK, let's close this for the time being, and continue the discussion at https://github.com/deepiksdev/tensorflow/issues/39
